guides/common/modules/proc_installing-the-satellite-server-packages.adoc

```diff
@@ -1,36 +1,21 @@
-ifdef::context[:parent-context: {context}]
-
 [id="Installing_Server_Packages_{context}"]
 = Installing {ProjectServer} Packages
 
 .Procedure
-
-ifdef::foreman-el,katello,satellite[]
-:context: el8
-:package-manager: dnf
-include::proc_installing-server-packages-el.adoc[]
-endif::[]
-
-ifdef::foreman-deb[]
-. Update package lists:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# apt-get update
-----
 . Update all packages:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {client-update-package-el8}
+ifdef::foreman-deb[]
+# apt-get update && apt-get upgrade
+endif::[]
+ifndef::foreman-deb[]
+# dnf update
```

Comment:
Can we use `package-update-project` here? 
https://github.com/theforeman/foreman-documentation/pull/2625/commits/68098c5099dcce432516d253eafb14b8ed414125#diff-e52c855d1d4e98ca6c2f2b52ee1b9307c5ce43cdd88d8b4aaa4613d18b33f75bR149

(If yes, same elsewhere)

Reply:
Yes, I agree to this. :+1: 

Can `client-install-package-*` be used instead? As we are installing packages on the host.

Reply:
Technically yes, but I would really like to differentiate between commands that run on Foreman Server and any managed hosts.

Reply:
Okay. So either way can be good. IMO we can keep this way instead of having another attribute. 👍